### ISSUE TITLE
incorrectly encodes line endings

### ISSUE THREAD

**Issue by garretwilson** [BrightIT/escape-utils#9]
I have a section of code I need to escape for HTML. (Actually I only need the XML escaping for `&lt;`, `&gt;`, etc.

````
<?xml version="1.0" encoding="UTF-8"?>
<user authenticated="true" sort="d">
  <name>Jane Doe</name>
  ...
````

I selected the text and used the the "Html Encode" command. The library encoded the line endings!!!!!!

````
&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;&#13;&#10;&lt;user authenticated=&quot;true&quot; sort=&quot;d&quot;&gt;&#13;&#10;  &lt;name&gt;Jane Doe&lt;/name&gt;&#13;&#10;  ...
````

Then I tried to use the "Html Encode Maintain Lines" command. Same thing. _The library encodes the line endings._

This is completely unexpected and counter-intuitive. How do I make it leave my line endings alone and merely encode special characters?

(P.S. You really need a "XML Encode" command, that only encodes the special XML characters. I don't need the em-dash character changed to `&mdash;`, for example---I'm using UTF-8 after all.)

**Comment by PiotrCzapla** [comment]
@garretwilson once you selected the lines split the selection to multiple selection using shift-cmd-l, then use Html Encode and it should give you what you want.
Here is what I've got:
```

&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;user authenticated=&quot;true&quot; sort=&quot;d&quot;&gt;
  &lt;name&gt;Jane Doe&lt;/name&gt;
  ...
```

**Comment by PiotrCzapla** [comment]
I'm not sure what has happened that the "Html Encode Maintain Lines stopped working :(, we will have a look.

**Comment by PiotrCzapla** [comment]
issue with "Html Encode Maintain Lines" was fixed here: 74e6ce98194375f2b7b485045c7dd8121818c7a9

**Comment by PiotrCzapla** [comment]
new release v0.5.3 was published. Thank you for reporting the issue.
